docs/usage/controls.md

```diff
@@ -105,3 +105,25 @@ You will have to use multiple instances of `Radio`. Each of those will itself yi
   <button type='submit'>Submit</button>
 </HeadlessForm>
 ```
+
+## Select
+
+Renders a native `<select>` dropdown. It will yield another `Option` component for the individual options, which receives a `@value` and the visible label as the block content, just like the native `<option>` element.
+
+```hbs preview-template
+<HeadlessForm as |form|>
+  <form.Field @name='country' as |field|>
+    <div class='my-2 flex flex-col'>
+      <field.Label>Country</field.Label>
+      <field.Select class='border rounded px-2' as |select|>
```

Comment:
**Question:** is there a way to switch the underlying tag (but keep the role) if necessary? I see some selects made with popovers and `ul/li` tag combinations.